### DISCUSSION TITLE
Safari iOS doesn't launch into AR directly when clicking on the AR button

### DISCUSSION THREAD

**Discussion by hinkelrex** [google/model-viewer#4421] [Q&A]
Hello, 

when I try to open the AR view of a model in safari, it doesn't open Quick View directly, but first redirects the user to the url of the model (see the attached image). Only when clicking that, Quick View is opened, which is a pretty bad user experience. 

Interestingly this doesn't happen on Google Chrome (iOS). There it starts Quick View directly. 

I honestly don't know what is causing this. I can see an error "Not found (url of the model)" in the safari console, but the URL should obviously be valid, since it redirects correctly and also works in other browsers, so I', kind of lost here. 

Does anyone have any help on this? 

![InkedIMG_0120](https://github.com/google/model-viewer/assets/120087179/7e7008fa-87a0-48d2-894b-1b3e6f693afd)

**Comment by samaneh-kazemi** [comment]
Hi, thanks for reaching out. Do you happen to have a glitch demo we could use for debugging? Also, which version of iOS are you using? There is an known issue #4421 for iOS17.
